docs/instrumentation-list.yaml

```diff
@@ -42,6 +44,8 @@ libraries:
       - com.typesafe.akka:akka-actor_2.13:[2.3,)
   alibaba:
   - name: alibaba-druid-1.0
+    description: |
+      The Druid instrumentation generates database connection pool metrics for druid data sources.
```

Comment:
Since besides alibaba druid there is also apache druid, that is a completely different project, perhaps we should start with `The Alibaba Druid instrumentation ...`